packages/fast-components-styles-msft/src/flipper/index.ts

```diff
@@ -22,37 +21,36 @@ const styles: ComponentStyles<FlipperClassNameContract, DesignSystem> = (
     config: DesignSystem
 ): ComponentStyleSheet<FlipperClassNameContract, DesignSystem> => {
     const designSystem: DesignSystem = withDesignSystemDefaults(config);
-
-    const backgroundColor: string = designSystem.backgroundColor;
     const direction: Direction = designSystem.direction;
-    const foregroundColor: string = ensureNormalContrast(
-        designSystem.contrast,
-        designSystem.foregroundColor,
-        designSystem.backgroundColor
-    );
-
-    const glyphColorHover: string = hoverContrast(config.contrast, foregroundColor);
 
     return {
         flipper: {
             width: "40px",
             height: "40px",
             margin: "0",
-            color: foregroundColor,
-            ...outlinePattern.rest,
+            color: neutralForegroundRest,
+            background: neutralFillStealthRest,
+            border: `${toPx(
+                designSystem.outlinePatternOutlineWidth
+            )} solid ${neutralOutlineRest(designSystem)}`,
             borderRadius: "50%",
             padding: "0",
             "&:hover": {
-                ...outlinePattern.hover,
+                border: `${toPx(
+                    designSystem.outlinePatternOutlineWidth
+                )} solid ${neutralOutlineHover(designSystem)}`,
                 "& $flipper_glyph": {
                     "&::before": {
-                        borderRightColor: glyphColorHover,
-                        borderTopColor: glyphColorHover,
+                        borderRightColor: neutralForegroundRest,
+                        borderTopColor: neutralForegroundRest,
                     },
                 },
             },
             ...applyFocusVisible({
-                ...outlinePattern.focus,
+                boxShadow: `0 0 0 1px ${designSystem.foregroundColor} inset`,
```

Comment:
`designSystem.foregroundColor` is being deprecated - is this for the focus indicator?

I think this is okay to leave for now, but if this is just for focus can you add TODOs as you're going? I've created a task to track the focus updates #1461 you can reference